src/main/java/propertyFiles/Bundle.properties

```diff
@@ -818,11 +818,11 @@ advanced.search.datasets.persistentId=Dataset Persistent ID
 advanced.search.datasets.persistentId.tip=The unique persistent identifier for a dataset, which can be a Handle or DOI in Dataverse.
 
 # search 
-search.datasets.literalquestion=Literal Question
-search.datasets.interviewinstructions=Interview Instructions
-search.datasets.postquestion=Post Question
-search.datasets.variableuniverse=Variable Universe
-search.datasets.variableNotes=Variable Notes
+search.datasets.literalquestion=Text of the actual, literal question asked.
+search.datasets.interviewinstructions=Specific instructions to the individual conducting an interview.
+search.datasets.postquestion=Text describing what occurs after the literal question has been asked.
+search.datasets.variableuniverse=The group of persons or other elements that are the object of research and to which any analytic results refer.
+search.datasets.variableNotes=For clarifying information/annotation regarding the parent element.
```

Comment:
The notes element can be used as a sub-element within many other elements. I think that's why [the description in the DDI documentation is so vague](https://ddialliance.org/Specification/DDI-Codebook/2.5/XMLSchema/field_level_documentation.html). The parent element here would be the variable, right? If so, could we edit the tooltip to read something like this?:

> For clarifying information/annotation regarding the variable.

Reply:
Thanks, updated in https://github.com/IQSS/dataverse/pull/6577/commits/f00348b162840ccae8071bb7ecba2c99dc305a32